data/cloudstack.yml

```diff
@@ -1,17 +1,16 @@
 versions:
-  main: 4.17.0.1
+  main: 4.17.1.0
   main_islts: yes
   main_docs_url: https://docs.cloudstack.apache.org/en/4.17.0.0/
   branch: 4.17
-  lts: 4.17.0.1
+  lts: 4.17.1.0
   lts_docs_url: https://docs.cloudstack.apache.org/en/latest/installguide/index.html
   ltsbranch: 4.17
   archives:
     - '4.17':
-       - 4.17.0.1
```

Comment:
@rohityadavcloud should we keep 4.17.0.1 release here?

Reply:
Yes, we should add both

Reply:
Fixed